### PR TITLE
PI-1939, PI-1940 - fix ui discrepancies on admin home and data sources page

### PR DESCRIPTION
**Pull request by montoresdennis**
Before:
image
image

After:
image
image

### CODE REVIEW

**Review by pierpuccini — APPROVED** [review]

**Comment by jazdw** [comment]
@montoresdennis here's another problem, these two top bars should be different colors (they are in the dark theme). (open the date bar to see them both)
image

The theme names are also inconsistent, can you rename them to `piMangoDark` and `piMangoLight` (camel case)

File: UI/web-src/ngMango/piMango.css [hunk 1]
```diff
@@ -85,6 +85,14 @@ md-content.md-pimangolight-theme .ma-ui-menu {
     color: var(--ma-primary-contrast);
 }
 
+md-list md-list-item.ma-list-selected button.md-pimangolight-theme {
+    color: white;
```
Comment (jazdw) [inline UI/web-src/ngMango/piMango.css:89]: Use a palette color if possible

**Review by jazdw — CHANGES_REQUESTED** [review]
See

**Comment by montoresdennis** [comment]
> @montoresdennis here's another problem, these two top bars should be different colors (they are in the dark theme). (open the date bar to see them both) image
> 
> The theme names are also inconsistent, can you rename them to `piMangoDark` and `piMangoLight` (camel case)

image

solved

**Comment by montoresdennis** [comment]
@jazdw @greychrist Ok, I discussed the coloring stuff with Rosbely and this is the final result
image

**Comment by greychrist** [comment]
I love this.  Thanks!

**Pull request merged**